<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bessel算法加速</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs.qrc">
    <normaloff>:/images/nku.png</normaloff>:/images/nku.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="imgs.qrc">:/images/nku.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>运算次数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>35</height>
              </size>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentText">
              <string>100</string>
             </property>
             <property name="maxVisibleItems">
              <number>10</number>
             </property>
             <item>
              <property name="text">
               <string>100</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>500</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10000</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_matlab_loc">
           <property name="text">
            <string>Matlab程序</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_data_loc">
             <property name="text">
              <string>数据位置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_gen_data">
             <property name="text">
              <string>生成数据</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_matlab_script">
           <property name="text">
            <string>Matlab脚本</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>160</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>加速算法</string>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>16</x>
              <y>46</y>
              <width>60</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>运行时间：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_accel_time">
            <property name="geometry">
             <rect>
              <x>78</x>
              <y>46</y>
              <width>96</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>16</x>
              <y>84</y>
              <width>60</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>运算次数：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_accel_counts">
            <property name="geometry">
             <rect>
              <x>78</x>
              <y>84</y>
              <width>96</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>16</x>
              <y>122</y>
              <width>60</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>单次时间：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_accel_once">
            <property name="geometry">
             <rect>
              <x>78</x>
              <y>122</y>
              <width>96</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_accel">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>运行加速算法</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_accel_result">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>查看加速结果</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>160</height>
            </size>
           </property>
           <property name="title">
            <string>Matlab</string>
           </property>
           <widget class="QLabel" name="label_19">
            <property name="geometry">
             <rect>
              <x>16</x>
              <y>46</y>
              <width>60</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>运行时间：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_matlab_time">
            <property name="geometry">
             <rect>
              <x>78</x>
              <y>46</y>
              <width>96</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_21">
            <property name="geometry">
             <rect>
              <x>16</x>
              <y>84</y>
              <width>60</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>运算次数：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_matlab_counts">
            <property name="geometry">
             <rect>
              <x>78</x>
              <y>84</y>
              <width>96</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_23">
            <property name="geometry">
             <rect>
              <x>16</x>
              <y>122</y>
              <width>60</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>单次时间：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_matlab_once">
            <property name="geometry">
             <rect>
              <x>78</x>
              <y>122</y>
              <width>96</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_matlab">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>运行Matlab算法</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_matlab_result">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>查看Matlab结果</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>160</height>
        </size>
       </property>
       <property name="title">
        <string>结果对比</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>30</y>
          <width>209</width>
          <height>144</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>加速比：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_result_ratio">
             <property name="minimumSize">
              <size>
               <width>96</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>加速算法精度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_result_1">
             <property name="minimumSize">
              <size>
               <width>96</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Matlab算法精度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_result_2">
             <property name="minimumSize">
              <size>
               <width>96</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_report">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>生成结果报告</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
